frameworks/Rust/thruster/thruster.dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM rust:1.26
```

Comment:
let's bump this to `rust:1.27` for the latest and greatest!

Reply:
I chose 1.26 to stay in sync with the other images to avoid compiler differences - I believe they're all bumped at once before a round is run.

Reply:
@whitfin FYI, we don't necessarily do that bump ourselves. If you'd like, you can bump it in all rust dockerfiles and assuming it doesn't break any of them, we'll merge that in. You don't need to keep it in sync though.

Reply:
@nbrady-techempower awesome - I'll probably file a separate one to bump all of them.